### PR TITLE
Standardize data point

### PR DESCRIPTION
**Pull request by skillitzimberg**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|   ✓ | :scroll: Docs              |

## Description
This PR applies the new format for URLs and other data points that change from one team to the next.

There were a couple of data points that hadn't been named and I just updated them to the new format: {cohort start date} is now {COHORT_START_DATE}.

In addition I updated the `PROJECT-BRIEF.md` to match the formatting of the advanced project's version. I noticed that the Shopping List's project brief is truncated, but it looks like the Orientation guides are taking over that role.

There are also a few edits for consistency between the Spanish and English versions of the Orientation documents.

### CODE REVIEW

File: ORIENTATION-en.md [hunk 1]
```diff
@@ -9,27 +8,28 @@ The goal of this project is to work together to build a “smart” shopping lis
...
 
-## How does it all work?  
+## How does it all work?
 
-The project is organized in GitHub as a set of [user stories](https://www.mountaingoatsoftware.com/agile/user-stories), each with a description of the desired functionality as well as [acceptance criteria](https://www.leadingagile.com/2014/09/acceptance-criteria/ (AC)) that describe how you know whether the task or story is complete. You can find the stories on [the project board]({link to the cohort repo project board}) on GitHub. This is referred to as the “backlog” (the collection of stories) and represents the work needed to complete the project. 
+The project is organized in GitHub as a set of [user stories](https://www.mountaingoatsoftware.com/agile/user-stories), each with a description of the desired functionality as well as [acceptance criteria](https://www.leadingagile.com/2014/09/acceptance-criteria/ 'AC') that describe how you know whether the task or story is complete. You can find the stories on [the project board]({PROJECT_BOARD_URL}) on GitHub. This is referred to as the “backlog” (the collection of stories) and represents the work needed to complete the project.
```
Comment (segdeha) [inline ORIENTATION-en.md]: [acceptance criteria](https://www.leadingagile.com/2014/09/acceptance-criteria/ (AC))

should be

    [acceptance criteria (AC)](https://www.leadingagile.com/2014/09/acceptance-criteria/)

File: PROJECT-BRIEF.md [hunk 2]
```diff
@@ -4,11 +4,11 @@
...
-- Site: {link to cohort live site}
-- Repo: {link to cohort repo}
-- Clone URL: `{clone URL}`
-- Issue list: {link to cohort repo issues}
-- Database: {link to cohort firebase database}
+- Site: {PRODUCTION_URL}
+- Repo: {REPO_URL}
+- Clone URL: {REPO_CLONE_URL}
+- Issue list: {PROJECT_BOARD_URL}
+- Google Cloud Console: {CONSOLE_URL}
```
Comment (segdeha) [inline PROJECT-BRIEF.md]: This should stay saying "Database" here. For the advanced project, we don't use Firestore, but rather Google Maps APIs, hence the different wording. Thanks!

**Review by segdeha — APPROVED** [review]
Just 2 small corrections, otherwise this looks great! Thanks so much for doing this, @skillitzimberg!!! Go ahead and merge it once you've made the 2 changes I specify

File: ORIENTATION-en.md (hunk 1 above)
Comment (skillitzimberg) [reply ORIENTATION-en.md]: I thought so too. Interestingly I saw that with the `AC` there, when you hover over the link `AC` is displayed as if it were a title attribute on an anchor tag. So I left it thinking it was intentional.

**Pull request merged**